internal/kafka/utils.go

```diff
@@ -120,6 +120,19 @@ func getCmkClusterPendingSize(cluster *cmkv2.CmkV2Cluster) int32 {
 	return -1
 }
 
+func getCmkMaxEcku(cluster *cmkv2.CmkV2Cluster) *int32 {
+	if isBasic(cluster) {
+		return cluster.Spec.Config.CmkV2Basic.MaxEcku
```

Comment:
```suggestion
		return cluster.Spec.GetConfig().CmkV2Basic.GetMaxEcku()
```
Let's use the get function here and return an int32 instead of a pointer so that we don't have to dereference it later.

We can return -1 for the default return instead of nil, and check that it isn't -1.

Reply:
That's a great point! My only concern is with Dedicated clusters, since they don’t have a MaxEcku field. If we switch to returning int32 with -1 as the default, it will show up as -1 when describing Dedicated clusters. To avoid that, we may still want to keep it as a pointer.